src/test/scala-2.11/markets/participants/MarketParticipantSpec.scala

```diff
@@ -55,7 +55,10 @@ class MarketParticipantSpec extends TestKit(ActorSystem("MarketParticipantSpec")
     val tradable = TestTradable("GOOG")
     val market = TestProbe()
     val markets = mutable.Map[Tradable, ActorRef](tradable -> market.ref)
-    val tickers = mutable.Map[Tradable, Agent[immutable.Seq[Tick]]](tradable -> Agent(immutable.Seq.empty[Tick]))
+
+    val initialTick = Tick(1, 1, 1, 1, timestamp())
+    val tickers = mutable.Map[Tradable, Agent[Tick]](tradable -> Agent(initialTick))
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [It's preferred to use immutable collections instead of mutable ones](https://www.codacy.com/app/davidrpugh/markets-sandbox/file/2168327547/issues/source?bid=2921361&fileBranchId=3162043#l60)